### PR TITLE
fw 2.16.1 (new formula)

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/fw.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  desc "Workspace productivity booster"
+  homepage "https://github.com/brocode/fw"
+  url "https://github.com/brocode/fw/archive/refs/tags/v2.16.1.tar.gz"
+  sha256 "2d77ce6673e48dd8f20ee42d8a64a0cbe648d51b837cb930b69c3707718659c2"
+  license "WTFPL"
+
+  depends_on "rust" => :build
+  depends_on "openssl@3"
+
+  uses_from_macos "zlib"
```
Comment (carlocab) [inline Formula/fw.rb:11]:
```
  fw
    * Dependency 'zlib' is provided by macOS; please replace 'depends_on' with 'uses_from_macos'.
  Error: 1 problem in 1 formula detected.
```
HI @Bo98

**Comment by chenrui333** [comment]
I think this is ready to merge?

**Comment by carlocab** [comment]
Yes, but can you file an issue at Homebrew/brew so it can be fixed?

**Comment by Bo98** [comment]
Ah sorry, missed that ping. Yeah should be ok to merge. Will take a look tonight.

**Comment by chenrui333** [comment]
> Yes, but can you file an issue at Homebrew/brew so it can be fixed?

totally, raised it, https://github.com/Homebrew/brew/issues/15645

**Review by p-linnane — APPROVED** [review]

**Pull request merged**